### DISCUSSION TITLE
Unable to open stderr buffer again if killed

### DISCUSSION THREAD

**Discussion by doolio** [joaotavora/eglot#1036] [Q&A] [answered]
Not sure if this is a bug so asking here first. If I kill (`C-x k`) the `*EGLOT stderr*`  buffer I cannot open it again with `M-x eglot-stderr-buffer` and see this error message:

> Wrong type argument: stringp, nil

If I kill the `*EGLOT events*` buffer I can only re-open it if in a buffer connected to the server but perhaps that is the expected. If I kill it and try to open it from a buffer not connected to the server I get the same error as above.

**Comment by joaotavora** [comment, marked as answer]
Sorry for the delay.  I think this is a bug, though not a serious one.  I've filed an issue with full repro: #1062
